Google Scholar.js

```diff
@@ -866,6 +886,8 @@ ItemFactory.prototype.getTitle = function () {
 	// Citelet parsing, step (4) [optional]
 	if (this.commaSplit) {
 		this.v.title = this.commaSplit.join(", ");
+	} else {
+		throw new Error("Parsing failed for case item \"" + this.citelet + "\"");
 	}
```

Comment:
Are you sure this is where the issue comes from? If title was never assigned, then Zotero would throw an error. Current behavior suggests that the translator is returning without calling `item.complete`